server/src/routes/api/v1/BookListRouter.js

```diff
@@ -25,12 +28,31 @@ BookListRouter.get("/:id", async (req, res) => {
   try {
     const { id } = req.params;
     const bookList = await BookList.query().findById(id);
-    const serializedBookList = await BookListSerializer.getSummary(bookList);
 
-    return res.status(200).json({ bookList: serializedBookList });
+    bookList.books = await bookList.$relatedQuery("books");
+    return res.status(200).json({ bookList });
   } catch (err) {
     return res.status(500).json({ err });
   }
 });
 
+bookBookListRouter.post("/", async (req, res) => {
+  const { body } = req;
+  const formInput = cleanUserInput(body);
+  const { name, imageUrl } = formInput;
+  const userId = req.user.id;
+
+  try {
+    const newBookList = await Book.query().insertAndFetch({ name, imageUrl, id, userId });
+    const serializedBookList = await BookListSerializer.getSummary(newBookList);
+    return res.status(201).json({ bookList: serializedBookList });
```

Comment:
hmmmm this code doesnt make sense:
- line 46 adds a new BOOK, but you call the variable `newBookList`
- line 47 uses a BookListSerializer, but the object you pass in (newBookList) actually contains a BOOK object 

what do you want to have happen here? it seems to be going in a few different directions, and Im not sure why you made this a nested router?